max_roll: .435332
max_pitch: .435332
max_yaw_rate: 3.1415
max_thrust: 74.0
mass: 3.81